bin/oq

```diff
@@ -26,6 +26,7 @@ if os.path.realpath(__file__) == '/opt/openquake/bin/oq':
     # https://docs.python.org/2/library/sys.html#sys.path
     sys.path.insert(1, '/opt/openquake/lib/python' + str(sys.version_info[0]) +
                     '.' + str(sys.version_info[1]) + '/site-packages')
+    os.environ['PYTHONPATH'] = ":".join(sys.path)
```

Comment:
On Windows we never enter that if, so this code will never executed (anyway, `os.environ` + `PYTHONPATH` works too, we use it in `.bat` entry points).